src/core/components/apps/app/AppView.js

```diff
@@ -1,50 +1,50 @@
-import React, { Component } from 'react'
-import deepEqual from 'deep-equal'
-import { Match, matchPattern } from 'react-router'
+import React, { Component } from 'react';
+import deepEqual from 'deep-equal';
+import { Match, matchPattern } from 'react-router';
 
-import KibanaApplicationRunner from './KibanaApplicationRunner'
-import KibanaApplicationBase from './KibanaApplicationBase'
+import AppWrapper from './AppWrapper';
```

Comment:
I think `Wrapper` is one of those names that doesn't provide any information, like `Manager`, `Data`, or `Obj`. Since the role of this is to adapt one set of lifecycle hooks (React) to another (apps), how about changing this to `Adapter`?